custom_endpoint/src/Controller/CustomEndpointController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\custom_endpoint\Controller;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\custom_endpoint\CurrentUserService;
+
+/**
+ * Custom Endpoint Controller class
+ */
+class CustomEndpointController extends ControllerBase {
+
+  public $currentUserService;
+
+  /**
+   * Custom Endpoint controller constructor
+   *
+   * @param Drupal\custom_endpoint\CurrentUserService
```

Comment:
the FQCN needs to start with the trailing `\` and point to the parameter which it defines, also the each parameter needs a description, so in this case:
```
* @param \Drupal\custom_endpoint\CurrentUserService $currentUser
*   The current user service.
```

Reply:
fixed